general-concepts/package-maintainers/text.xml

```diff
@@ -5,6 +5,63 @@
 
 <body>
 
+<section>
+<title>Maintainers</title>
+<body>
+
+<p>
+Package maintainers are responsible for ebuild maintenance tasks —
+bumping them to new upstream releases, updating as necessary wrt
+policies, replying to bugs, etc. They are listed in package's
+<uri link="::ebuild-writing/misc-files/metadata/">metadata.xml</uri>
+file. When assigning bugs, the first maintainer listed becomes the bug's
+assignee and the remaining maintainers are added to CC.
+</p>
+
+<p>
+Package maintainers have authority over their maintained packages.
+Unless specified otherwise, you should request maintainers' approval
+before committing to their packages. The exceptions to this rule are
+trivial changes implied by your own actions, e.g. dependency updates
+as a result of package moves.
+</p>
+
+<p>
+In case of maintainer's inactivity, a customary tool of maintainer
+timeout may be used. This timeout can be invoked if none of
+the package's maintainers reply to a patch within 4 weeks of it being
```

Comment:
The other section (see URL above) says "a soft limit of 2 to 4 weeks" so this should say the same.